file-updates/release-resources/application.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Application
+metadata:
+  name: fileupdatestest
```

Comment:
should probably include namespace, no?

Reply:
since we are using kubeconfig files, the namespace is defined by them

Reply:
alright. the ones in the fbc-test dir have namespaces and the release CRs in this dir have namespaces too, so I thought it would be nice for consistency. Either way is fine though